lib/rabbit_feed/event.rb

```diff
@@ -2,42 +2,52 @@ module RabbitFeed
   class Event
     include ActiveModel::Validations
 
-    attr_reader :schema, :payload
-    validates_presence_of :schema, :payload
+    attr_reader :schema, :payload, :metadata
+    validates :metadata, presence: true
```

Comment:
Yes, that is a good point. I don't want to go too overboard with validation though, as I want it to be simple to create sample events for consumer tests. Thoughts?